bciers/apps/compliance/src/app/components/compliance-summaries/ComplianceSummariesPage.tsx

```diff
@@ -14,28 +17,48 @@ export default async function ComplianceSummariesPage({
   const { reporting_year: reportingYear } = await getReportingYear();
   const initialData = await fetchComplianceSummariesPageData(searchParams);
   const frontEndRole = await getSessionRole();
+  const currentDate = new Date(); // Current date
+  const dueDate = new Date(reportingYear + 1, 10, 30, 23, 59, 59); // November 30 of the reporting year + 1
   const isAllowedCas = [
     FrontEndRoles.CAS_DIRECTOR,
     FrontEndRoles.CAS_ANALYST,
     FrontEndRoles.CAS_ADMIN,
     FrontEndRoles.CAS_VIEW_ONLY,
   ].includes(frontEndRole);
+  const isAllowedIndustryUser = [
+    FrontEndRoles.INDUSTRY_USER,
+    FrontEndRoles.INDUSTRY_USER_ADMIN,
+  ];
```

Comment:
This is missing the `.includes(frontEndRole)`, so `isAllowedIndustryUser` is always going to be true. Or could use `!isAllowedCas` in the alert conditional instead